tools/operator/launcher/k8s/extendNetwork.go

```diff
@@ -0,0 +1,135 @@
+package k8s
+
+import (
+	"fmt"
+
+	"github.com/pkg/errors"
+
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/client-go/kubernetes"
+
+	"github.com/hyperledger/fabric-test/tools/operator/fabricconfig"
+	"github.com/hyperledger/fabric-test/tools/operator/launcher/nl"
+	"github.com/hyperledger/fabric-test/tools/operator/logger"
+	"github.com/hyperledger/fabric-test/tools/operator/networkspec"
+	//"github.com/hyperledger/fabric-test/tools/operator/paths"
+)
```

Comment:
Remove commented line, blanks lines, and sort

Reply:
fixed